orderportal/html/home.html

```diff
@@ -213,7 +200,7 @@
 	  {% end %}
 	</table>
 	{% else %}
-	<i>Currently none. Check again later.</i>
+	<em>Currently none. Check again later.</em>
```

Comment:
Like `<b>` vs. `<strong>` - using `<em>` _(emphasis)_ instead of `<i>` is best practice.